css/css-color/rec2020-001.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: rec2020</title>
+<link rel="author" title="Sam Weinig" href="mailto:weinig@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-color-4/#valdef-color-rec2020">
+<link rel="match" href="greensquare-ref.html">
+<meta name="assert" content="rec2020 with no alpha">
+<style>
+    .test { background-color: red; width: 12em; height: 12em; }
+    .test { background-color: color(rec2020 0.234171 0.429825 0.084989); } /* green (sRGB #008000) converted to rec2020 */
```

Comment:
For this one (I checked all the previous conversions, btw and they all agreed) I get
[color(rec2020 0.235202 0.431704 0.085432)](https://colorjs.io/apps/convert/?color=%23008000&precision=6) so let me check my implementation and check against the CSS Color 4 sample code, too.

Reply:
![image](https://user-images.githubusercontent.com/2506926/106410868-8a914380-644c-11eb-85cf-e47c36aaec3f.png)

I get the same with the CSS Color 4 sample code.

Reply:
The code I used to compute this was:

```c
#import <Foundation/Foundation.h>
#import <CoreGraphics/CoreGraphics.h>

int main(int argc, const char * argv[]) {
        CGFloat components[4] = { 0, .5, 0, 1.0f };

        CGColorSpaceRef sRGB_colorSpace = CGColorSpaceCreateWithName(kCGColorSpaceSRGB);
        CGColorRef inputColor = CGColorCreate(sRGB_colorSpace, components);
        CFRelease(sRGB_colorSpace);

        CGColorSpaceRef ITUR_2020_colorSpace = CGColorSpaceCreateWithName(kCGColorSpaceITUR_2020);
        CGColorRef outputColor = CGColorCreateCopyByMatchingToColorSpace(ITUR_2020_colorSpace, kCGRenderingIntentDefault, inputColor, NULL);
        CFRelease(sRGB_colorSpace);

        const CGFloat* outputComponents = CGColorGetComponents(outputColor);

        NSLog(@"Result: %f %f %f", outputComponents[0], outputComponents[1], outputComponents[2]);

        CFRelease(inputColor);
        CFRelease(outputColor);
    }
    return 0;
}
```

Reply:
I get a bit closer to yours if I explicitly use RGB(0, 0.501961, 0)

```c
#import <Foundation/Foundation.h>
#import <CoreGraphics/CoreGraphics.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        CGFloat components[4] = { 0, .501961, 0, 1.0f };

        CGColorSpaceRef sRGB_colorSpace = CGColorSpaceCreateWithName(kCGColorSpaceSRGB);
        CGColorSpaceRef ITUR_2020_colorSpace = CGColorSpaceCreateWithName(kCGColorSpaceITUR_2020);

        CGColorRef inputColor = CGColorCreate(sRGB_colorSpace, components);
        CGColorRef outputColor = CGColorCreateCopyByMatchingToColorSpace(ITUR_2020_colorSpace, kCGRenderingIntentDefault, inputColor, NULL);

        const CGFloat* outputComponents = CGColorGetComponents(outputColor);

        NSLog(@"Result: %f %f %f", outputComponents[0], outputComponents[1], outputComponents[2]);

        CFRelease(sRGB_colorSpace);
        CFRelease(ITUR_2020_colorSpace);
        CFRelease(inputColor);
        CFRelease(outputColor);
    }
    return 0;
}
```

`Result: 0.235443 0.431843 0.085691`

Reply:
Both end up round tripping to the same 8 bit sRGB, so I didn't notice the difference in the test.